WalletWasabi/Exceptions/TransientException.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WalletWasabi.Exceptions
+{
+	/// <summary>
+	/// Infrastructure or optimistic conflict exception that will be fixed when retried
```

Comment:
e.g. when database if offline. you can retry the transaction in the same way as in case of conflict.

Reply:
Period would against English grammar. That is not an sentence because it doesn't contain a verb. English grammar says you should put periods at the end of sentences. That comment is just a long name. You don't put periods after names 🤷‍♂️